arkouda/dataframe.py

```diff
@@ -83,15 +84,17 @@ class GroupBy:
         The column name(s) associated with the aggregated columns.
     as_index : bool (default=True)
         If True the grouped values of the aggregation keys will be treated as an index.
-
+    dropna : bool (default=True)
+        If True, and the groupby keys contain NaN values,
+        the NaN values together with the corresponding row will be dropped.
+        Otherwise, the rows corresponding to NaN values will be kept.
```

Comment:
I'm not sure if this should be listed as an attribute since we never call anything like `self.dropna`

Reply:
I fixed this.  Thanks!